m3u_copy.py

```diff
@@ -31,4 +33,4 @@ def m3u_copy(source, destination):
                     return
 
 
-m3u_copy("source/file1.m3u", "destination/")
+m3u_copy(sys.argv[1], sys.argv[2])
```

Comment:
You should also add a check if two parameters are provided. If script is called without them, a usage example should be printed.